### PR TITLE
RuboCop linting

### PR DESCRIPTION
**Pull request by olleolleolle**
In order to pass RuboCop checks, this PR 

- Adds minimal documentation to classes that lack it.
- Uses public_send
- Configures RuboCop: Enforce double_quotes 
- Adds a TODO file to RuboCop config
- Refactors a few methods (Extract Method refactorings)

### CODE REVIEW

File: lib/how_is/report.rb [hunk 1]
```diff
@@ -4,12 +4,15 @@
 require "pathname"
 
 class HowIs
+  # Error class
```
Comment (olleolleolle) [inline lib/how_is/report.rb]: This one could echo the one about import.

File: spec/how_is_spec.rb [hunk 2]
```diff
@@ -109,21 +109,21 @@
   end
 
   # Disable 'cop' that is violated by every .generate_frontmatter() calls.
```
Comment (duckinator) [inline spec/how_is_spec.rb]: while you're working on this file, could you change "calls" to "call", here? also I guess replace `'cop'` with `rule` or something.

**Comment by duckinator** [comment]
You're doing excellent work on this! 😄

File: spec/how_is_spec.rb (hunk 2 above)
Comment (olleolleolle) [inline spec/how_is_spec.rb]: Thanks, good find.

**Comment by duckinator** [comment]
🎉 🎉 🎉  awesome, thanks!

**Pull request merged**